### PR TITLE
Feature/repairswagger: Adding powershell cmdlet for repair alert

### PR DESCRIPTION
**Pull request by sumitabarahmand**
## Description

Adding powershell cmdlet for repair alert
Spec PR: https://github.com/Azure/azure-rest-api-specs/pull/4084
SDK PR: https://github.com/Azure/azure-sdk-for-net/pull/5000

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by sumitabarahmand** [comment]
@bganapa  and @deathly809 
Spec: https://github.com/Azure/azure-rest-api-specs/pull/4084
SDK: https://github.com/Azure/azure-sdk-for-net/pull/5000

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by deathly809** [comment]
Will review after SDK is merged.

**Comment by bganapa** [comment]
@sumitabarahmand  please update this PR once the SDK  PR 
https://github.com/Azure/azure-sdk-for-net/pull/5000 is merged and published.

**Comment by sumitabarahmand** [comment]
The SDK is published: https://azuresdkci.westus2.cloudapp.azure.com/view/NetCore-SDK/job/NetCore-SDK-Publish/180/

**Comment by deathly809** [comment]
@maddieclayton Seems the website tests failed,  

```
Skipped  Microsoft.Azure.Commands.Websites.Test.ScenarioTests.WebAppBackupRestoreTests.TestRestoreDeletedWebAppToExisting
[xUnit.net 00:00:13.89]     Microsoft.Azure.Commands.Websites.Test.ScenarioTests.WebAppBackupRestoreTests.TestGetDeletedWebApp [FAIL]
```

and for default

```
0.9999002s ✘  Microsoft.Azure.Commands.Websites.Test.ScenarioTests.WebAppBackupRestoreTests.TestGetDeletedWebApp
```

**Comment by bganapa** [comment]
ps-sign https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1556/console

File: src/StackAdmin/Azs.InfrastructureInsights.Admin/Tests/src/Alert.Tests.ps1 [hunk 1]
```diff
@@ -225,5 +226,22 @@ InModuleScope Azs.InfrastructureInsights.Admin {
                 }
             }
         }
+
+        it "TestRepairAlert" -Skip:$('TestRepairAlert' -in $global:SkippedTests) {
```
Comment (deathly809) [inline src/StackAdmin/Azs.InfrastructureInsights.Admin/Tests/src/Alert.Tests.ps1]: You should add a test for alerts that cannot be repaired and repairs on alerts that don't even exist.

**Comment by bganapa** [comment]
@sumitabarahmand 
can you please edit these files for module version update
https://github.com/Azure/azure-powershell/blob/preview/src/StackAdmin/Azs.InfrastructureInsights.Admin/Module/Azs.InfrastructureInsights.Admin/Azs.Infrastructureinsights.Admin.psd1#L20
https://github.com/Azure/azure-powershell/blob/preview/src/StackAdmin/AzureStack/AzureStack.psd1#L62

otherwise it looks good to me

File: src/StackAdmin/Azs.InfrastructureInsights.Admin/Tests/src/Alert.Tests.ps1 (hunk 1 above)
Comment (deathly809) [reply src/StackAdmin/Azs.InfrastructureInsights.Admin/Tests/src/Alert.Tests.ps1]: No, we use these tests in VaaS for validation.  These are required.

**Comment by bganapa** [comment]
ps-sign - https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1563/console

**Comment by maddieclayton** [comment]
@bganapa Can you get @deathly809 to approve as well? His request for changes is blocking merge.

**Comment by bganapa** [comment]
@maddieclayton Thanks for looking at this PR. @deathly809 is OOF and his concern of adding a negative test case has been addressed now. The previous ps-sign failed due to module version mismatch..
a new one is running now https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1565/

**Comment by deathly809** [comment]
@maddieclayton Could you please merge this?  Everything looks fine to us.

**Pull request merged**